app/assets/javascripts/application.js

```diff
@@ -17,6 +19,7 @@
 //= require hidden_values
 
 $(document).on('ContentLoad', function(){onContentLoad()});
+Turbolinks.enableProgressBar();
```

Comment:
This call has to go at the beginning of ```onContentLoad``` in order to work, it won't load on it's own here.